CHANGES.md

```diff
@@ -81,6 +81,7 @@ To be released.
  -  Improved performance of `Block<T>.Deserialize()`.  [[#1116]]
  -  `Swarm<T>` became not to fill states from trusted peers, because now states
     can be validated rather than trusted due to MPT.  [[#1117]]
+ -  `HashDigest<SHA256>` became serializable.  [[#1125]]
```

Comment:
We address issue number if it exists.
```suggestion
 -  `HashDigest<SHA256>` became serializable.  [[#795], [#1125]]
```

Additionally, please add link to those before [line #90](https://github.com/planetarium/libplanet/pull/1125/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R90) and below [line #107](https://github.com/planetarium/libplanet/pull/1125/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R107)

```
[#795]: https://github.com/planetarium/libplanet/issues/795
[#1052]: https://github.com/planetarium/libplanet/pull/1052
```

```
[#1119]: https://github.com/planetarium/libplanet/pull/1119
[#1125]: https://github.com/planetarium/libplanet/pull/1125
```